### PR TITLE
Image component created

### PR DESCRIPTION
**Pull request by Lokeshkhati**
image

image

Please review this Image component

### CODE REVIEW

File: css/components/image.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.img-round {
+  height: 12rem;
+  width: 12rem;
+  border-radius: 50%;
+  object-fit: cover;
+}
+.img-responsive {
+  width: 100%;
```
Comment (rahulcs754) [inline css/components/image.css]: you should use here max-width, not width only.

**Review by rahulcs754 — COMMENTED** [review]
I checked your code 
that is good 
just a small change and read about what happens if you apply max-width

File: css/components/image.css (hunk 1 above)
Comment (Lokeshkhati) [reply css/components/image.css]: Thanks for your kind suggestion

**Review by ttahm3d — APPROVED** [review]
Good enough to merge.

**Pull request merged**